frontend/src/components/LightingTalkList/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import { observer } from 'mobx-react-lite';
+import { lightingTalkStore, uiStore } from '../../stores';
+import imgTitle from '../../assets/images/hack-news.png';
+import { LightingTalk, VoteLightingTalkRequest } from '../../models';
+import LightingTalkListItem from '../LightingTalkListItem';
+import LightingTalkEditor from '../LightingTalkEditor';
+import './index.scss';
+
+type Props = {};
+
+const LightingTalkList: React.FC<Props> = observer((props) => {
+  // Toggle editor show/hidden
+  const [hasLogined, _] = useState(uiStore?.hasLogined);
+  const [showEditor, setShowEditor] = useState(false);
+
+  // Add new topic
+  const onAddTopic = () => {
+    setShowEditor(true);
+  };
+
+  // On talk selected
+  const onSelectItem = (item: LightingTalk) => {
+    item.showContent = !item.showContent;
+    lightingTalkStore.setLightingTalk([...lightingTalkStore.lightingTalk]);
+  };
+
+  // On Vote
+  const onVote = async (item: LightingTalk) => {
+    const request = {
+      id: item.id,
+    } as VoteLightingTalkRequest;
+    lightingTalkStore
+      .voteLightingTalk(request)
```

Comment:
Mixed style of Promise and async / await

Reply:
Yes ,you're right 
'async' is unnecessary.